doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -0,0 +1,136 @@
+                              .size = 0x2000UL,
+                              .type = PCIBAR_MEM32
+                              },
+                              [1] = {
+      -                       .base = 0xb3f53000UL,
+      +                       .base = 0x91537000UL,
+                              .size = 0x100UL,
+                              .type = PCIBAR_MEM32
+                              },
+                              [5] = {
+      -                       .base = 0xb3f52000UL,
+      +                       .base = 0x91536000UL,
+                              .size = 0x800UL,
+                              .type = PCIBAR_MEM32
+                              },
+                       }
+                     }
+                  },
+               }
+        };
+
+3. Configure ``.bootargs`` for vm kernel command linue. A default command line is configured by default.
+
+   Here we just configure ``root=`` option to help kernel locate the right rootfs and keep the other options
+   as default.
+
+   .. code-block:: none
+
+      @@ -171,7 +180,8 @@ struct vm_description_array vm_desc_partition = {
+                                      .start_hpa = 0x100000000UL,
+                                      .mem_size = 0x20000000UL, /* uses contiguous memory from host */
+                                      .vm_vuart = true,
+      -                               .bootargs = "root=/dev/sda rw rootwait noxsave maxcpus=2 nohpet console=hvc0 \
+      +                               .bootargs = "root=PARTUUID=<Your rootfs PARTUUID> rw rootwait noxsave maxcpus=2 nohpet console=hvc0 \
+                                                    console=ttyS0 no_timer_check ignore_loglevel log_buf_len=16M \
+                                                    consoleblank=0 tsc=reliable xapic_phys",
+                                      .vpci_vdev_array = &vpci_vdev_array1,
+
+4. Build ACRN with the above changes.
+
+   .. code-block:: none
+
+      $ make PLATFORM=sbl
+
+5. Install ubuntu on your UP2 and add one menuentry in ``<$efi_root>/grub/grub.cfg``
+
+   .. code-block:: none
+
+      menuentry 'ACRN Partition Mode' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+              recordfail
+              load_video
+              gfxmode $linux_gfx_mode
+              insmod gzio
+              insmod part_gpt
+              insmod ext2
+
+              echo 'Loading partition mode hypervisor ...'
+              multiboot /acrn.32.out
+              module /bzImage
+      }
+
+    Note: Make sure SATA and USB related modules are configured as "Y" of bzImage.
+
+6. Copy ``acrn.32.out`` and bzImage to ``<$efi_root>/``
+
+7. Insert Udisk or/and SATA SSD passthroughed to vms with rootfs installed into UP2 and then reboot board to 
+   EFI shell. Then execute the following command to load acrn.
+
+   .. code-block:: none
+
+      $ EFI\ubuntu\grubx64.efi
+
```

Comment:
Add one step to instruct the user to select the `ACRN Partition Mode` menu entry

Reply:
Sure, will do.